src/cryptoadvance/specter/util/reflection.py

```diff
@@ -64,8 +64,8 @@ def get_package_dir_for_subclasses_of(clazz):
     elif clazz.__name__ == "Service":
         return str(
             Path(
-                import_module("cryptoadvance.specter.services").__file__
-            ).parent.resolve()
+                import_module("cryptoadvance.specterext.devhelp").__file__
```

Comment:
why devhelp?

Reply:
I'll add a comment:
```
        # here, we'd like to know the Path of the namespace module cryptoadvance.specterext
        # but that fails because you cannot import namespace-modules.
        # So we take a module that we know exists and take its parent.
```

Reply:
OK, not trivial :-)